### PR TITLE
Remove cAndCwithStickyE project files

### PR DESCRIPTION
**Pull request by mnwhite**
Per team discussions, projects like cAndCwithStickyE should live only in their own REMARK, not duplicated in the package distribution of HARK.  This commit removes the project files for StickyE.

### CODE REVIEW

**Comment by llorracc** [comment]
I will do this when we have verified that everything works properly for a fresh install.  Probably nearly there now, but this [issue](https://github.com/llorracc/cAndCwithStickyE/issues/1) needs to be fixed (on the cAndCwithStickyE-Latest private repo).  When that is done, I'll run my scripts again to generate the public version.  

I'd also like to do a global search in HARK, DemARK, and REMARK for things that might break.

**Comment by sbenthall** [comment]
This is the only repository in the `econ-ark` organization with a reference to `HARK/cAndCwithStickyE`:

https://github.com/econ-ark/econ-ark-make/search?q=cAndCwithStickyE&unscoped_q=cAndCwithStickyE
It is a checklist with no functionality.

The other references to cAndCwithStickyE are to the `llorracc/cAndCwithStickyE` version, which is a REMARK:
https://github.com/econ-ark/REMARK/search?q=cAndCwithStickyE&unscoped_q=cAndCwithStickyE
https://github.com/econ-ark/REMARK-make/search?q=cAndCwithStickyE&unscoped_q=cAndCwithStickyE

Or the paper:
https://github.com/econ-ark/econ-ark.org/search?q=cAndCwithStickyE&unscoped_q=cAndCwithStickyE

https://github.com/econ-ark/PARK-make/search?q=cAndCwithStickyE&unscoped_q=cAndCwithStickyE
https://github.com/econ-ark/HARK-make/search?utf8=%E2%9C%93&q=cAndCwithStickyE&type=
https://github.com/econ-ark/PLUTARK/search?q=cAndCwithStickyE&unscoped_q=cAndCwithStickyE

In other words, I could find no reason to think removing these HARK files would break downstream code.

**Comment by llorracc** [comment]
This is done in #442
